values.yaml

```diff
@@ -459,6 +459,52 @@ server:
     # to the service.
     annotations: {}
 
+    # Enables a headless service that contains the active pod only
+    active:
+      # clusterIP controls whether a Cluster IP address is attached to the
```

Comment:
Maybe add a comment about these values defaulting to the `values.server.service` values if missing, except `annotations` (which will be appended)?